src/core/core.config.js

```diff
@@ -31,20 +37,36 @@ function axisFromPosition(position) {
   }
 }
 
-export function determineAxis(id, scaleOptions) {
-  if (id === 'x' || id === 'y' || id === 'r') {
+export function determineAxis(id, ...scaleOptions) {
+  if (idMatchesAxis(id)) {
     return id;
   }
+  for (let i = 0, n = scaleOptions.length; i < n; i++) {
```

Comment:
because the i/n are not really needed, remind me if there was a reason not to use `for (const opts of scaleOptions)`?

Reply:
No special reason. I have read that for is faster than for..of but I can change it tomorrow. Now I dont have my laptop with me

Reply:
@etimberg @kurkle do you want I change the for cycle? Asking you because you approved and I'll dismiss your approvals. Let me know

Reply:
Let's go ahead and update it while we're here and then I can re-approve